developer/openshift/gitops/argocd/pipeline-service/metrics-exporter/servicemonitor.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# for RHTAP, the pipeline-service monitor is defined in infra-deployments, but we define here in our developer folder (vs. the operator folder) to define this out of dev_setup.sh
```

Comment:
The RHTAP def of the service monitor in infra deployments is over at https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/prometheus/base/servicemonitors/pipeline-service.yaml#L31-L44

You's see some diffs as of the writing of this comment, namely the `namespaceSelector` and the use of `honorLabels`.

If we merge these changes, and then bump the pipeline-service level in infra-deployments, and then when I test, I still cannot access the metrics from the openshift console, I will be changing the data at https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/prometheus/base/servicemonitors/pipeline-service.yaml#L31-L44 to exactly match what I have in my dev folder here.